CleverPush/CleverPush/Source/CPAppBanner.m

```diff
@@ -6,25 +6,31 @@ @implementation CPAppBanner
 - (id)initWithJson:(NSDictionary*)json {
     self = [super init];
     if (self) {
-        self.id = [json objectForKey:@"_id"];
-        self.channel = [json objectForKey:@"channel"];
-        self.name = [json objectForKey:@"name"];
-        self.HTMLContent = [json objectForKey:@"content"];
-        self.contentType = [json objectForKey:@"contentType"];
-        if ([json objectForKey:@"testId"] != nil) {
-            self.testId = [json objectForKey:@"testId"];
-        }
-        if ([[json objectForKey:@"type"] isEqual:@"top"]) {
+        self.id = [json valueForKey:@"_id"];
```

Comment:
Please use `stringForKey`
we should * never * use `valueForKey` as it might throw an exception. `objectForKey` or `stringForKey` will not do that

Reply:
Same for all the ones below